quinn-proto/src/connection/send_buffer.rs

```diff
@@ -416,6 +450,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
Should this still be cleaned up? If we really want to ignore the test we should at least write a reason why we're doing so.

Reply:
This test just won't work anymore since small writes are coalesced. I think I will try making the parts bigger (>MAX_COMBINE), then we should be able to see the old behaviour.

Reply:
I renamed and modified the test.